services/intersection-api/jpo-conflictvisualizer-api/src/main/java/us/dot/its/jpo/ode/api/ConflictApiApplication.java

```diff
@@ -0,0 +1,76 @@
+package us.dot.its.jpo.ode.api;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+import us.dot.its.jpo.ode.api.asn1.DecoderManager;
+import us.dot.its.jpo.ode.api.models.postgres.derived.UserOrgRole;
```

Comment:
Issue (blocking): When I try to build this locally (or run it in my IDE), I get a compilation error. Should we add this package to the repo or add it to our pom.xml as a dependency?

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project jpo-conflictvisualizer-api: Compilation failure: Compilation failure: 
[ERROR] /home/orion/repos/CDOT-CV/jpo-cvmanager/services/intersection-api/jpo-conflictvisualizer-api/src/main/java/us/dot/its/jpo/ode/api/ConflictApiApplication.java:[12,54] package us.dot.its.jpo.ode.api.models.postgres.derived does not exist
[ERROR] /home/orion/repos/CDOT-CV/jpo-cvmanager/services/intersection-api/jpo-conflictvisualizer-api/src/main/java/us/dot/its/jpo/ode/api/ConflictApiApplication.java:[13,39] cannot find symbol
[ERROR]   symbol:   class PostgresService
[ERROR]   location: package us.dot.its.jpo.ode.api.services
[ERROR] /home/orion/repos/CDOT-CV/jpo-cvmanager/services/intersection-api/jpo-conflictvisualizer-api/src/main/java/us/dot/its/jpo/ode/api/ConflictApiApplication.java:[28,16] cannot find symbol
[ERROR]   symbol:   class PostgresService
[ERROR]   location: class us.dot.its.jpo.ode.api.ConflictApiApplication
```

Reply:
This has been resolved